example/goodrx-cds-price-comparison/README.md

```diff
@@ -0,0 +1,120 @@
+# GoodRx Prices as a Decision Support API
+
+## Overview
+
+Transparency on drug pricing changes the [decisions](https://pubmed.ncbi.nlm.nih.gov/11025790/) [doctors](https://pubmed.ncbi.nlm.nih.gov/29255097/) [make](https://pubmed.ncbi.nlm.nih.gov/29321043/). Good data on what prescriptions cost _while they are prescribing them_ is both something (1) doctors say they want and (2) doctors who have the data have been shown to have reduced per-patient perscription costs than those who do not. Meanwhile, open standards have emerged ([CDS Hooks](https://cds-hooks.hl7.org/)) as a way of getting decision support to doctors directly inside of the EHR (i.e. patient specific recommendations on drug pricing).
+
+In this example. we introduce the use of third party consumer pricing data ([GoodRx](https://www.goodrx.com/)) over a decision support API provided by the Sero toolkit to deliver drug price to doctors while they are making the prescription. Sero can be used to make decision support APIs and SMART on FHIR apps.
+
+https://www.loom.com/share/05034bda49e240daa2ee353505a0d3cc
+
+## Walkthrough
+
+CDS Hooks is an open decision support API that `@sero.run/sero` implements support for. As part of HL7, and closely related to FHIR, CDS Hooks is the only open standard for 3rd party developers ____. [Spec, Internal, Reference conformance]
```

Comment:
Not fully written, want to keep it short and basically just link to other things (including to parts of the guide)